src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int NAME = 0;
+    private static final int NUMBER = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        List<String> fileContent = readFile(fromFileName);
+        String report = getResult(fileContent);
+        writeToFile(toFileName, report);
+    }
+
+    private static List<String> readFile(String fromFileName) {
```

Comment:
Static methods are in general a bad practice. Let’s better leave this method on instance level (not class).